extensions/ticktick/src/create.tsx

```diff
@@ -74,6 +77,16 @@ export default function TickTickCreate() {
 
   const onListChange: NonNullable<Form.Dropdown.Props["onChange"]> = useCallback((newValue) => {
     LocalStorage.setItem("defaultAddList", newValue);
+    setProjectId(newValue);
+  }, []);
+
+  const onTitleBlur: NonNullable<Form.TextField.Props["onBlur"]> = useCallback(async (newValue) => {
+    const title = newValue.target.value;
+    if (title && environment.canAccess(AI)) {
+      const toast = await showToast(Toast.Style.Animated, "🧠 Guessing project...");
```

Comment:
debounce guessing project when on change and cancel debounce on blur and guessing project immediately will be better

Reply:
That would make sense, I like the idea 🙂

Reply:
Great idea! I put a 500ms debounce. We might hit the rate limit if we type slowly and add multiple tasks in a row, but in normal usage it works fine to me. 🙌

Reply:
I guess we can resolve this